build-scratch

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
-SHA=`git rev-parse HEAD`
-docker build -t $SHA . &&
-    docker run --rm --entrypoint tar $SHA -c /usr/bin/ct Dockerfile.build-scratch | docker build -f Dockerfile.build-scratch --tag coreos/ct -
+
+set -e
+VERSION=$(./git-version)
+docker build -t $VERSION .
+docker run --rm --entrypoint tar $VERSION -c /usr/bin/ct Dockerfile.build-scratch \
+    | docker build -f Dockerfile.build-scratch --tag quay.io/coreos/ct:$VERSION -
+docker tag quay.io/coreos/ct:$VERSION quay.io/coreos/ct:latest-dev
```

Comment:
It's the sha1/tag based on git-version, there isn't any hash unless I'm misunderstanding. This just tags it with the sha, and the latest-dev tag.